Sources/SuperwallKit/Logger/LogScope.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  File.swift
+//  
+//
+//  Created by Yusuf Tör on 22/11/2022.
+//
+
+import Foundation
+
+/// The possible scope of logs to print to the console.
+public enum LogScope: String, Sendable {
```

Comment:
Does this need to be public?

Reply:
Yeah you can set logging and its scopes in `SuperwallOptions`

Reply:
Could you also make it an `Int` enum so its compatible with objc?

Reply:
Done